src/CoreBundle/Command/ActiveCronjobCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Chameleon System (https://www.chameleonsystem.com).
+ *
+ * (c) ESONO AG (https://www.esono.de)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ChameleonSystem\CoreBundle\Command;
+
+use ChameleonSystem\CoreBundle\CronJob\CronjobEnablingServiceInterface;
+use ChameleonSystem\CoreBundle\Exception\CronjobHandlingException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ActiveCronjobCommand extends Command
```

Comment:
I agree to your thought - "running" instead of "active" is clearer and unambiguous.

Commands should have a "command-like" imperative name - maybe something like `GetCronjobRunStateCommand`?

Reply:
Using "GetCronjobsStateCommand"

Reply:
Sounds good.